webclient/static/js/leaflet-search.js

```diff
@@ -130,15 +130,64 @@ L.Control.Search = L.Control.extend({
 		this._countertips = 0;		//number of tips items
 		this._recordsCache = {};	//key,value table! to store locations! format: key,latlng
 		this._curReq = null;
+		this._url={}
+		this._url["KU"]={name:this.options.translations.KatastralniUzemi,
+			urlSuggestions:'https://ags.cuzk.cz/arcgis/rest/services/RUIAN/Vyhledavaci_sluzba_nad_daty_RUIAN/MapServer/exts/GeocodeSOE/tables/7/suggest?maxSuggestions=100&outSR={"latestWkid":5514,"wkid":102067}&f=json',
```

Comment:
Nedáme adresu https://ags.cuzk.cz/arcgis/rest/services/ někam jako konstantu nebo do nastavení aplikace? Kdyby byla potřeba změna, tak ať to staží změnit na jednom místě

Reply:
Beru to tak, že jsem to předělal pro naše speciální potřeby, s našimi adresami. To vyhledávání má nejenom speciální adresy, ale i speciální parametry a těch nastavení je tam mnoho, že naopak je nejvhodnější to mít na jednom místě přímo v leaflet-search.js

Reply:
@jhavrlant Ok, necháme to, jak to je.